defaults/main.yml

```diff
@@ -10,11 +10,11 @@ aiidalab_server_crypt_key: "{{ lookup('password', '/tmp/crypt_key chars=ascii_le
 # Whether to create new users on first login, if they do not exist
 aiidalab_server_create_users: false  # can be true or false
 
-# Set to true in order to build aiidalab docker image locally
-aiidalab_server_build_locally: false
-
-# Tag of docker stack, see https://hub.docker.com/r/aiidalab/aiidalab-docker-stack
-aiidalab_server_docker_stack: latest  # or 'develop'
+# Manage AiiDAlab Docker image.
+aiidalab_server_build_locally: false # Set 'true' to download the Git repository and build the image locally. Set 'false' to pull the image from DockerHub.
+aiidalab_image_name: aiidalab/aiidalab-docker-stack # Used when pulling the image from DockerHub.
+aiidalab_image_git_repository: https://github.com/aiidalab/aiidalab-docker-stack.git # Used when pulling the image from the Git repository.
+aiidalab_image_version: latest # or 'develop', or '21.12.0'.
```

Comment:
The reason for using the `aiidalab_server` prefix is that ansible variables are in a global namespace, and so it can be helpful to prefix variable names with the name of the role (here: aiidalab-server).

The remaining name changes are fine with me.

Reply:
Good point, thanks. I will adapt it.

Reply:
done